mods/ra/maps/soviet-06a/soviet06a.lua

```diff
@@ -87,7 +87,7 @@ Trigger.OnKilled(Apwr2, function(building)
 	BaseApwr2.exists = false
 end)
 
-Trigger.OnKilled(Dome, function()
+Trigger.OnKilledOrCaptured(Dome, function()
 	player.MarkCompletedObjective(sovietObjective2)
 	Media.PlaySpeechNotification(player, "ObjectiveMet")
```

Comment:
I suggest expanding this to
```
	Trigger.AfterDelay(DateTime.Seconds(2), function()
		Media.PlaySpeechNotification(player, "ObjectiveMet")
	end)
```
so that the "Structure captured" notification doesn't overlap with "Objective met".